src/dotnet/APIView/APIViewWeb/APIViewWeb.csproj

```diff
@@ -58,6 +58,7 @@
     <PackageReference Include="NuGet.Protocol" Version="6.10.0" />
     <PackageReference Include="Octokit" Version="13.0.1" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.5.0" />
+    <PackageReference Include="System.Text.Json" VersionOverride="[8.0.5, )" />
```

Comment:
Why are you adding this version override? Also I don't think VersionOverride works in this repo, it would just be version if you need to directly reference this.